commands/01-mod.js

```diff
@@ -733,8 +733,16 @@ Module
       if (num > 0) {
         while (num > 0) {
           let deleting = Math.min(num, 50)
-          await channel.bulkDelete(deleting);
-          num -= deleting;
+          deleted = await channel.bulkDelete(deleting, filterOld=true);
+          num -= deleted.size;
+          if (deleted.size != deleting)
+            break;
+        }
+        if (num > 0) {
+          let msgsToDelete = await channel.messages.fetch({limit=num, before=msg.id});
```

Comment:
Similar to line 736, it's a matter of formatting here.
```suggestion
          let msgsToDelete = await channel.messages.fetch({limit: num, before: msg.id});
```